backend/src/routes.js

```diff
@@ -298,12 +297,12 @@ function createJWTs(id, role) {
 }
 
 // function to check if user is Admin or not
-function checkAdmin(auth) {
+function isAdmin(auth) {
     const accessToken = auth.split(' ')[1];
     const payload = jwt.decode(accessToken);

```

Comment:
You don't need to decode the access token again. This has already been done by the express-jwt middleware.
This middleware has exposed `req.user` which contains the payload. So you can check the role directly there.

Reply:
a ok.. I didn't knew it..
so it's no need to use the function isAdmin too, right??